qa/L0_dyna_implicit_state/test.sh

```diff
@@ -34,7 +34,7 @@ if [ -z "$REPO_VERSION" ]; then
     echo -e "\n***\n*** Test Failed\n***"
     exit 1
 fi
-if [ ! -z "$TEST_REPO_ARCH" ]; then
+if [ ! -z "$TEST_REPO_ARCH" ] && [[ "$REPO_VERSION" == *"$TEST_REPO_ARCH"* ]]; then
```

Comment:
Suppose it's behavior should be:
```suggestion
if [ ! -z "$TEST_REPO_ARCH" ] && [[ "$REPO_VERSION" != *"$TEST_REPO_ARCH"* ]]; then
```

Reply:
I think you're right, sorry I missed this. Let me fix it.